docs/source/external_exporters.rst

```diff
@@ -139,8 +139,13 @@ when you are at the directory under which you wish to build your ``mypackage`` p
 
 .. important::
     You should not publish this package without adding content to your ``LICENSE.md`` file.
-    ``nbconvert`` follows the Jupyter Project convention of using a Modified BSD License (also 
-    known as New or Revised or 3-Clause BSD). You are not required to use this License.
+    For example, ``nbconvert`` follows the Jupyter Project convention of using a Modified BSD 
+    License (also known as New or Revised or 3-Clause BSD).
+    For a guide on picking the right license for your use case,
+    please see `choose a license<choosealicense.com>`.
```

Comment:
I think this needs an underscore at the end to make it a link. Are you set up to build the docs? If not,  Matthias can help you, but it should be a matter of:

- Install the necessary packages
- Run `make html` in the right directory